python/ray/dashboard/client/src/pages/metrics/Metrics.tsx

```diff
@@ -304,6 +304,34 @@ const DATA_METRICS_CONFIG: MetricsSectionConfig[] = [
         title: "Task Submission Backpressure Time",
         pathParams: "orgId=1&theme=light&panelId=37",
       },
+      {
+        title: "(p00) Task Completion Time",
+        pathParams: "orgId=1&theme=light&panelId=38",
+      },
+      {
+        title: "(p05) Task Completion Time",
+        pathParams: "orgId=1&theme=light&panelId=39",
+      },
+      {
+        title: "(p50) Task Completion Time",
+        pathParams: "orgId=1&theme=light&panelId=40",
+      },
+      {
+        title: "(p75) Task Completion Time",
+        pathParams: "orgId=1&theme=light&panelId=41",
+      },
+      {
+        title: "(p90) Task Completion Time",
+        pathParams: "orgId=1&theme=light&panelId=42",
+      },
+      {
+        title: "(p99) Task Completion Time",
+        pathParams: "orgId=1&theme=light&panelId=44",
+      },
+      {
+        title: "(p100) Task Completion Time",
```

Comment:
We don't need that many panels -- we can get rid of p0, p05, p90 (just keep 4 -- p50/75/99/100)